docs/src/main/sphinx/installation/query-resiliency.rst

```diff
@@ -0,0 +1,120 @@
+===================================
+Improve query processing resilience
+===================================
+
+You can configure Trino to be more resilient against failures during query
+processing by enabling fault-tolerant execution. This allows Trino to handle
+larger queries such as batch operations without worker node interruptions
+causing the query to fail.
+
+When configured, the Trino cluster buffers data used by workers
+during query processing. If a worker node fails for any reason, such as a
+network outage or running out of available resources, the coordinator issues the
+data to an available worker that can pick up query processing, allowing query
+processing to continue.
+
+Architecture
+------------
+
+The coordinator node uses a configured exchange manager service that buffers
+data during query processing in an external location, such as an S3 object
+storage location. Worker nodes send data to the buffer as they execute their
+query tasks.
+
+Best practices and considerations
+---------------------------------
+
+A fault-tolerant cluster is best suited for large batch queries. Fault-tolerant
+execution of a high volume of short-running may experience increased latency. As
```

Comment:
Not an native... but phrasing that "execution may experience increased latency" sounds awkward. It is rather an user of Trino who experience certain behaviour (like increased latency)

Reply:
How's this for a rewrite?